src/invoice/mod.rs

```diff
@@ -96,6 +97,33 @@ impl Invoice {
         version_compare(self.bindle.id.version(), requirement)
     }
 
+    /// Check whether a group by this name is present.
+    pub fn has_group(&self, name: &str) -> bool {
+        let empty = vec![];
```

Comment:
This can be `Vec::with_capacity(0)` to avoid an allocation but not necessary to fix